.github/workflows/stale.yml

```diff
@@ -18,8 +18,8 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         days-before-stale: 180
         days-before-close: 180
-        stale-issue-message: "Thank you for your contribution!\n\nUnfortunately, this issue hasn't received much attention lately, so it is labeled as 'stale.'\n\nIf no additional action is taken, this issue will be automatically closed in 180 days."
-        stale-pr-message: "Thank you for your contribution!\n\nUnfortunately, this pull request hasn't received much attention lately, so it is labeled as 'stale.'\n\nIf no additional action is taken, this pull request will be automatically closed in 180 days."
+        stale-issue-message: "Thank you for your contribution!\n\nUnfortunately, this issue hasn't received much attention lately, so it is labeled as 'stale.'\n\nIf no additional action is taken, this issue will be automatically closed in 180 days.\n\nIf you want to participate in our prioritization meetings or be notified when support sprints are happened, you can sign up the support sprint notification emails [here](https://public.govdelivery.com/accounts/USDOIGS/signup/39118).\n\nRead more about our processs [here](https://astrogeology.usgs.gov/docs/how-to-guides/software-management/software-support/)"
```

Comment:
s/happened/happening/

Read me about our process here. What process specifically?

Reply:
From the top of the doc linked: 

> This document provides information about the software support process, including scheduling, project management tools and practices, issue solicitation, issue selection, and prioritization.

If the problem is with that doc, we can change it in that repo.